app/services/setting_registry.rb

```diff
@@ -44,6 +44,16 @@ def []=(name, value)
 
   def load
     # add() all setting definitions
+    load_definitions
+
+    # load all db values
+    load_values
+
+    # if create_missing create missing values in DB
+    # not needed now as old load mechanism takes care of that
```

Comment:
This is a bit hard to parse for me. What is the old load mechanism?

Reply:
`Setting.descendants.load_defaults`

Reply:
The way we do things now - create all the settings.
This is a note to keep in mind that if we would like to still have all the key/value pairs in db, this would be the place to create those :)

Reply:
I think this would be better, but I still can't understand the `if` part.
```suggestion
    # if create_missing creates missing values in the DB
    # not needed now as the old load mechanism takes care of that
```

And I would not use `old load mechanism` but rather describe it. Or define a method `old_load_mechanism` and call that.

What I'm mostly trying to say is that this comment created more confusion on my side than that it helped.

Reply:
Is it ok with you if I do this in the DSL PR, so we fix nightlies?

Reply:
Sounds good.